data/lib/src/network/linshare_http_client.dart

```diff
@@ -135,4 +157,13 @@ class LinShareHttpClient {
       return MailingListAutoCompleteResultDto.fromJson(map);
     }
   }
+
+  WorkGroupNodeDto _convertToWorkGroupNodeChild(Map<String, dynamic> nodeChildJson) {
+    final isDocument = (nodeChildJson['type'] as String) == WorkGroupNodeType.DOCUMENT.toString();
+    if (isDocument) {
+      return WorkGroupDocumentDto.fromJson(nodeChildJson);
+    } else { // is FOLDER
```

Comment:
I don't think we need to comment `// is FOLDER` here. Code is clear enough

Reply:
and for me, I hate if-else in functional programming convention

Reply:
Fixed in. Plz help me verify